tests/contrib/google_genai/utils.py

```diff
@@ -36,6 +36,23 @@
     ),
 )
 
+MOCK_GENERATE_CONTENT_RESPONSE_WITH_REASONING = types.GenerateContentResponse(
+    candidates=[
+        types.Candidate(
+            content=types.Content(
+                role="model",
+                parts=[
+                    types.Part.from_text(text="Let me think about this..."),
```

Comment:
Question - do we already capture reasoning messages from google genai? If not, could we do a follow up PR to do a best effort at capturing reasoning messages?